app/src/main/java/org/p2p/wallet/deeplinks/AppDeeplinksManager.kt

```diff
@@ -53,12 +54,18 @@ class AppDeeplinksManager(
                 val data = intent.data ?: return
 
                 val isValidScheme = context.getString(R.string.app_scheme) == data.scheme
+                val isTransferScheme = context.getString(R.string.transfer_app_scheme) == data.host
                 when {
                     isValidScheme && DeeplinkUtils.isValidOnboardingLink(data) -> {
                         rootListener?.triggerOnboardingDeeplink(data)
                     }
-                    context.getString(R.string.transfer_app_scheme) == data.scheme -> {
-                        transferPendingDeeplink = data
+                    isTransferScheme -> {
+                        val deeplink = SendViaLinkWrapper(data.toString())
+                        val isExecuted = rootListener?.parseTransferViaLink(deeplink) ?: false
+                        if (!isExecuted) {
+                            // postpone deeplink execution until app will be ready
+                            transferPendingDeeplink = data
```

Comment:
maybe you can put deeplink insude transferPendingDeeplink instead of data?

Reply:
I didn't get your idea, let's dm